### ISSUE TITLE
DO NOT USE WalletGenerator code or the WalletGenerator.net website --- They are compromised.

### ISSUE THREAD

**Issue by jeremyers1** [walletgeneratornet/WalletGenerator.net#290]
I used WalletGenetor.net to create some paper wallets. All my crypto was stolen a few days later. 

The site and code is compromised. DO NOT USE!

**Comment by jeremyers1** [comment]
Yes. And with the CyberCrimes division of the FBI.

**Comment by HenryEnrique** [comment]
Same thing happened to me but thanks to Blackhathacker111 who helped me get recovered my lost funds,message him on telegram his telegram username is https://t.me/Blackhathacker110

**Comment by HenryEnrique** [comment]
> Yes. And with the CyberCrimes division of the FBI.

I had lost a lot of coins in some scam case just like this thanks to the best hacker in the world who helped me recover all my funds,message him on telegram his telegram username is https://t.me/Blackhathacker110

**Comment by furkan3d** [comment]
Yes, Metamask has flagged this site as deceptive, I hope your loss is not too great. 
It further emphasises the importance of building without an internet connection and without using open source code.
__Evet,_ Metamask bu siteyi aldatıcı olarak işaretlemiş, umarım kaybınız büyük değildir. 
İnternet bağlantısı olmadan ve açık kaymak kod kullanmadan oluşturmanın önemini dahada değerlendiriyor._
